src/app/layout/layout.module.ts

```diff
@@ -30,8 +30,8 @@ import { RawDataModule } from '../modules/raw-data/raw-data.module';
     NgxSpinnerModule,
     ClipboardModule,
     UserIdleModule.forRoot({
-      idle: environment.userIdle,
-      timeout: environment.userIdle,
+      idle: 60,
+      timeout: 30,
```

Comment:
Was the intention to hardcode these? My first thought is that using the environment is better